src/main/sensors/gyro_init.c

```diff
@@ -223,33 +223,33 @@ static void dynLpfFilterInit()
     } else {
         gyro.dynLpfFilter = DYN_LPF_NONE;
     }
-    gyro.dynLpfMin = gyroConfig()->dyn_lpf_gyro_min_hz;
-    gyro.dynLpfMax = gyroConfig()->dyn_lpf_gyro_max_hz;
-    gyro.dynLpfCurveExpo = gyroConfig()->dyn_lpf_curve_expo;
+    gyro.dynLpfMin = gyroConfig()->gyro_lpf1_dyn_min_hz;
+    gyro.dynLpfMax = gyroConfig()->gyro_lpf1_dyn_max_hz;
+    gyro.dynLpfCurveExpo = gyroConfig()->gyro_lpf1_dyn_expo;
 }
 #endif
 
 void gyroInitFilters(void)
 {
-    uint16_t gyro_lowpass_hz = gyroConfig()->gyro_lowpass_hz;
+    uint16_t gyro_lowpass1_init_hz = gyroConfig()->gyro_lpf1_static_hz;
```

Comment:
I think it's better to keep the proposed name ruling here as well and rename the local variable 
```cpp
gyro_lowpass1_init_hz
```
to
```cpp
gyro_lpf1_init_hz
```
even though it's just a local variable.

Reply:
thanks.  done.

I'll do the blackbox changes later, don't want to delay this getting merged.